src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -668,9 +661,8 @@ export class MatFormField
       ((iconPrefixContainer ? iconPrefixContainerWidth - WRAPPER_HORIZONTAL_PADDING : 0) +
         textPrefixContainerWidth);
 
-    // Update the transform the floating label to account for the prefix container. Note
-    // that we do not want to overwrite the default transform for docked floating labels.
-    floatingLabel.style.transform = `${FLOATING_LABEL_DEFAULT_DOCKED_TRANSFORM} translateX(${labelHorizontalOffset}px)`;
+    // Update the transform x of the floating label to account for the prefix container.
+    floatingLabel.style.transform = `var(--mat-mdc-form-field-label-transform, translateX(${labelHorizontalOffset}px)`;
```

Comment:
Should this be?

```suggestion
    floatingLabel.style.transform = `var(--mat-mdc-form-field-label-transform) translateX(${labelHorizontalOffset}px)`;
```

Reply:
Actually, nevermind. I follow the logic. Just confused because the `translateY` docked transform is missing now? is that expected?

Reply:
Whoops, I think I deleted that when I was trying to figure out how to do the override. I'll add it back. I'll also add a comment explaining what this is doing, since its a little confusing